python/ray/_private/workers/default_worker.py

```diff
@@ -240,7 +240,7 @@
         else:
             object_spilling_config = {}
         external_storage.setup_external_storage(
-            object_spilling_config, node.session_name
+            object_spilling_config, node._node_id, node.session_name
```

Comment:
Add a public `@property` `node_id` to expose it instead of directly using `_node_id`